src/Mvc/global.json

```diff
@@ -1,8 +0,0 @@
-{
-    "sdk": {
-        "version": "3.0.100-preview-009750"
-    },
-    "msbuild-sdks": {
-        "Internal.AspNetCore.Sdk": "3.0.0-build-20181114.5"
-    }
-}
```

Comment:
upgrade deps deleted the global.json. Is that right?

Reply:
verified with nate offline, don't need the global.json. It exists at the root of the repo